src/main.ts

```diff
@@ -161,6 +161,8 @@ export async function run(): Promise<void> {
 
     core.setOutput('paths-output-file', pathsOutputFile)
     core.saveState('paths-output-file', pathsOutputFile)
+  } else if (filePatterns && !pathsOutput) {
+    core.warning('No match found for specified patterns. Ensure that subdirectory patterns a prefixed with "**/". See: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet')
```

Comment:
🚫 **[eslint]** <[prettier/prettier](https://github.com/prettier/eslint-plugin-prettier#options)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Replace `'No·match·found·for·specified·patterns.·Ensure·that·subdirectory·patterns·a·prefixed·with·"**/".·See:·https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet'` with `⏎······'No·match·found·for·specified·patterns.·Ensure·that·subdirectory·patterns·a·prefixed·with·"**/".·See:·https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet'⏎····`
```suggestion
    core.warning(
      'No match found for specified patterns. Ensure that subdirectory patterns a prefixed with "**/". See: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet'
    )
```